docs/lang/articles/misc/debugging.md

```diff
@@ -39,11 +39,26 @@ def inside_taichi_scope():
     v = ti.Vector([3, 4])
     print('v =', v)
     #=> v = [3, 4]
+
+    ray = ti.Struct({
+    	"ori": ti.Vector([0.0, 0.0, 0.0]),
+    	"dir": ti.Vector([0.0, 0.0, 1.0]),
+    	"len": 1.0
+		})
+    # print(ray)
+    # Print a struct directly in Taichi-scope has not been supported yet
+    # Instead, use:
+    print('ray.ori = ', ray.ori, ', ray.dir = ', ray.dir, ', ray.len = ', ray.len)
```

Comment:
```suggestion
    print('ray.ori =', ray.ori, ', ray.dir =', ray.dir, ', ray.len =', ray.len)
```
Would that result in two spaces if you leave a space after `=`?

Reply:
Nice catch!